create-variable/app.js

```diff
@@ -0,0 +1,2 @@
+// Create a variable named 'myName'
+var myName = "Paul Banjawan";
```

Comment:
Since this variable has not changed its value, you should use `const` instead of `var`.
```suggestion
const myName = "Paul Banjawan";
```

Reply:
hi @xianvalencia ,  this has already been addressed on my latest commit, thank you

Reply:
hi @xianvalencia , this has already been addressed on my latest commit, thank you